docs/README.rst

```diff
@@ -90,7 +90,7 @@ General
 |                       |         |          |                                  | to poll the cluster for its             |                |
 |                       |         |          |                                  | node members.                           |                |
 +-----------------------+---------+----------+----------------------------------+-----------------------------------------+----------------+
-| python-basedir        | string  | Optional | /app/python                      | Path to python utilities                |                |
+| python-basedir        | string  | Optional | n/a                              | Path to python utilities                |                |
```

Comment:
Is this something that users don't need to configure? If so, maybe we should consider taking this row out.

Reply:
Ya this is something that they do not need to configure, but can optionally configure if they want to use their own python bigipconfigdriver.py instead of the one that is pip installed with the controller. So I'll defer to you whether this row needs to be deleted or not.

Reply:
Let's leave it in, then, just make sure it's clear what the intended use is (i.e., what you just explained to me).